libs/feature/search/src/lib/favorites/favorite-star/favorite-star.component.html

```diff
@@ -10,10 +10,4 @@
     (newValue)="toggleFavorite($event)"
     [disabled]="loading || (isAnonymous$ | async)"
   ></gn-ui-star-toggle>
-  <span
-    class="align-text-top ml-1.5"
-    data-test="favorite-count"
-    *ngIf="!displayCount"
-    translate="datahub.record.addToFavorites"
```

Comment:
Maybe this translation key could be kept as a tooltip? It is a useful indication IMO (and probably required in terms of accessibility)

Reply:
It was actually not displayed anymore before this PR, so I've left it out for now. Displaying it as a simple tooltip could create a conflict with the "login message tooltip", displaying it in a tippy tooltip might be too intrusive and not necessarily improve accessibility.